### PR TITLE
Adding token balances script

### PR DESCRIPTION
**Pull request by Vasilije1990**
I've checked that:

* [ ] the query produces the intended results
* [ ] the folder name matches the schema name
* [ ] the schema name exists in Dune
* [ ] views are prefixed with `view_`, functions with `fn_`.
* [ ] the filename matches the defined view, table or function and ends with .sql
* [ ] each file has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`

### CODE REVIEW

File: ethereum/token_balances/token_balances.sql [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+
+DO $$
+DECLARE
+   hour_   text;
+   arr varchar[];
+BEGIN
+  select array(select generate_series('2021-03-21', '2021-03-23', '1 hour'::interval)::timestamptz)
+    into arr;
+   FOREACH hour_  IN   array arr
```
Comment (mewwts) [inline ethereum/token_balances/token_balances.sql:26]: Can this `DO` just be a `SELECT token_balances.insert_hourly(start_ts, end_ts)`?

**Review by mewwts — COMMENTED** [review]
Nice!

File: ethereum/token_balances/token_balances.sql [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+		, "historical_balances" as (
+			select
+				s.address as address
+				, s.contract_address as token_address
+				, s.rawAmount
+		    from sandbox.token_balances_proposal_1 s
+		    join "transfer_events" te
+		    on te.address = s.address
+		    and s.contract_address = te.token_address
+		    where s.ts < hour_::timestamptz
```
[the remark this answers is not in the document]
Comment (Vasilije1990) [reply ethereum/token_balances/token_balances.sql:62]: Yup, makes sense. The filter is too broad and the query would get slower over time.

File: ethereum/token_balances/token_balances.sql (hunk 1 above)
Comment (Vasilije1990) [reply ethereum/token_balances/token_balances.sql:26]: Makes sense, I thought along the same lines.

**Pull request closed without merging**